main.go

```diff
@@ -14,6 +11,10 @@ import (
 	"reflect"
 	"strings"
 	"time"
+
+	"github.com/zenazn/goji/bind"
+	"github.com/zenazn/goji/web"
+	"github.com/zenazn/goji/web/middleware"
```

Comment:
what's the logic here? this has oscillated before and i'd like to keep it from doing so in the future 😕